gremlin-scala/src/main/scala/gremlin/scala/GremlinScala.scala

```diff
@@ -954,16 +954,14 @@ class GremlinScala[End](val traversal: GraphTraversal[_, End]) {
     GremlinScala[Edge, Labels](traversal.bothE(labels: _*))
 
   /** may be used together with `from` / `to`, see TraversalSpec for examples */
-  def addE(label: String, properties: KeyValue[_]*)(
-      implicit ev: End <:< Vertex): GremlinScala.Aux[Edge, Labels] = {
+  def addE(label: String, properties: KeyValue[_]*): GremlinScala.Aux[Edge, Labels] = {
     val newTrav = properties.foldLeft(traversal.addE(label)) { (trav, prop) =>
       trav.property(prop.key.name, prop.value)
     }
     GremlinScala[Edge, Labels](newTrav)
   }
 
-  def addE(label: StepLabel[Vertex], properties: KeyValue[_]*)(
-      implicit ev: End <:< Vertex): GremlinScala.Aux[Edge, Labels] =
+  def addE(label: StepLabel[Vertex], properties: KeyValue[_]*): GremlinScala.Aux[Edge, Labels] =
```

Comment:
I found the test for `def addE(label: String,..` but not one for `addE(label: StepLabel[Vertex],..`

Is it worth adding a test for the addE StepLabel variant also, or would that be overkill?

Reply:
you're right, i was just being lazy here. good thing that you brought that up, the StepLabel should be `StepLabel[Edge]` here and not `StepLabel[Vertex]`. fixed